pkg/task/task.go

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
As I stated in
https://github.com/intelsdi-x/snap/pull/951#issuecomment-223271285
I did not manage to figure out where to properly religieux cate this code
Therefore I took'this wild guess and was expecting to get advices on where to put it so as to comply with snap/golang code best practices